pallets/bitacross-pallet/src/lib.rs

```diff
@@ -46,15 +32,33 @@ pub mod pallet {
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
 		// some extrinsics should only be called by origins from TEE
 		type TEECallOrigin: EnsureOrigin<Self::RuntimeOrigin>;
+		// origin to manage authorized delegatee list
+		type RelayerAdminOrigin: EnsureOrigin<Self::RuntimeOrigin>;
```

Comment:
It's called `RelayerAdminOrigin` but it's not to set relayers directly, instead, it sets the admin who can manage the relayers.

I suggest the name `SetAdminOrigin`

Reply:
done